components/install/installSection.js

```diff
@@ -29,8 +32,81 @@ const steps = [
   }
 ]
 
+const InstallInstructions = ({ token }) => (
+  <Card
+    shadowSz='lg'
+    maxW='55rem'
+    margin='0 auto 3rem'
+    textAlign='left'
+  >
+    <Text marginBottom='3rem'>
+        Copy and paste the applicable command in your shell to continue:
+    </Text>
+    <InstallCommands token={token} />
+  </Card>
+)
+
+InstallInstructions.propTypes = {
+  token: PropTypes.string.isRequired
+}
+
+const PostInstallInstructions = ({ noAds }) => {
+  const topText = noAds ? "There's nothing else you need to do." : "Let's try it out!"
+  return (
+    <Card
+      id='post-install'
+      shadowSz='lg'
+      maxW='55rem'
+      margin='0 auto 3rem'
+      textAlign='left'
+    >
+      <Text marginBottom='1rem'>
+        Great! {topText}
+      </Text>
+      <Text marginBottom='1rem'>
+        Flossbank wraps package managers (currently only <code>npm</code> and <code>yarn</code>).
+        This means that when you run your package manager to install a package, Flossbank will automatically be invoked.
+      </Text>
+      {noAds ? (
+        <Text marginBottom='1rem'>
+          Next time you run <code>npm</code> or <code>yarn</code>, Flossbank will silently determine the dependency tree of the packages installed and allocate your donation accordingly.
```

Comment:
we're going to have to remember to update this as we roll out more wrapped managers

Reply:
for sure